crates/ty_python_semantic/resources/mdtest/pep695_type_aliases.md

```diff
@@ -313,6 +313,26 @@ static_assert(is_subtype_of(Bottom[JsonDict], Bottom[JsonDict]))
 static_assert(is_subtype_of(Bottom[JsonDict], Top[JsonDict]))
 ```
 
+
+### Cyclic defaults
```

Comment:
This is a bit too advanced Python typing for me 😅 

I think it would be good to have some `reveal_type` or other assertion in this test but I'm not sure what that would be.